service/replication.go

```diff
@@ -278,6 +278,16 @@ func (s *service) CreateStorageProtectionGroup(ctx context.Context,
 	}, nil
 }
 
+// TODO: implement
+func (s *service) DeleteLocalVolume(ctx context.Context,
+	req *csiext.DeleteLocalVolumeRequest) (*csiext.DeleteLocalVolumeResponse, error) {
+	ctx, log, _ := GetRunIDLog(ctx)
+
+	log.Info("!!! Deleting Remote Volume !!!")
+
+	return &csiext.DeleteLocalVolumeResponse{}, nil
+}
+
```

Comment:
Is this method doing something? Stubs that are not yet implemented should throw some type of not implemented exception or log an error.

Reply:
The method is not doing anything in this library. It is mainly for compatibility at this point while we implement the functionality in csi-powerstore. 
Updated the function to return 'nil' response and an error regarding the unimplemented state.

Reply:
If we return a failure at this point in any of these repos, it will cause a regression-- current code does not fail, it simply doesn't delete the backend volume. All we can do right now is log an error. 

PowerScale in particular, though, should log that no action is being taken and that the backend folder will be deleted upon the next sync. It works a little differently from the other drivers since it's purely file-based storage.